src/Raven.Server/Documents/Indexes/IndexStore.cs

```
@@ -437,9 +445,16 @@ public Index GetIndex(string name)
 
                 var command = PutAutoIndexCommand.Create((AutoIndexDefinitionBase)definition, _documentDatabase.Name);
 
-                var (etag, _) = await _serverStore.SendToLeaderAsync(command);
+                var (etag, _) = await _serverStore.SendToLeaderAsync(command);      
+                try
+                {
+                    await _documentDatabase.RachisLogIndexNotifications.WaitForIndexNotification(etag, timeoutForIndexNotification);
 
-                await _documentDatabase.RachisLogIndexNotifications.WaitForIndexNotification(etag);
+                }
+                catch (TimeoutException)
```

Why is ignoring the exception a good idea?